akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -425,6 +427,38 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>generate-protobuf-endpoints</id>
+            <activation>
+                <file>
+                    <exists>src/main/protobuf</exists>
```

Comment:
I thought the directory was usually `proto`? maybe it can be either? I think we use to have `proto` in [samples](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/java-protobuf-eventsourced-counter/src/main/proto)?

Reply:
Ah, I'll change, IntelliJ suggests both (and we probably pick either up in Akka gRPC) but we indeed use `proto` Akka gRPC samples so let's go with that.

Reply:
For the record `protobuf` is the default in sbt/Scala projects